src/olympia/shelves/tests/test_serializers.py

```diff
@@ -307,8 +307,28 @@ def test_tag_token_substitution(self):
         assert data['title'] == {'en-US': 'Title for foo'}
         assert data['footer']['text'] == {'en-US': 'footer with foo'}
 
+        # Check this works when locale is requested we don't have - i.e. value is None
```

Comment:
Nit: This wording of this comment is a bit confusing. Did you have a superfluous or missing word in here?

Reply:
How about "Check this works when we don't have the requested locale"?